src/main/java/apoc/load/LDAPResult.java

```diff
@@ -0,0 +1,11 @@
+package apoc.load;
+
+import java.util.Map;
+
+public class LDAPResult {
```

Comment:
Or are there other useful columns that makes sense to add? like data provenance ones?